tests/sandbox/create_message_batches/test_performance.py

```diff
@@ -36,7 +44,17 @@ def test_create_messages_large_valid_payload(nhsd_apim_proxy_url):
 @pytest.mark.sandboxtest
 def test_create_messages_large_invalid_payload(nhsd_apim_proxy_url):
     """
-    .. py:function:: Test large (50k) invalid payload
+    .. py:function:: Scenario: A user submitting a request with a \
+        request body containing 50,000 invalid messages receives a 400 status code \
+        and response body containing the first 100 instances of errors
+
+        | **Given** the user provides a message body of 50,000 invalid messages
+        | **When** the user submits a request
+        | **Then** the response is a 400 error
```

Comment:
...containing 100 errors

Reply:
and takes less than 29 seconds - we're probably after the speed here rather than the fact it contains all the errors